docs/usage/usage-guide.md

```diff
@@ -196,7 +196,7 @@ const todosReducer = createReducer([], {
         // Can still return an immutably-updated value if we want to
         return state.filter( (todo, i) => i !== action.payload.index)
     }
-}
+})
```

Comment:
This is no typo. The `)` closes the argument list of the `createReducer()` call.

Reply:
So I just added it

Reply:
Oops, sorry, I read the diff the wrong way around. 😅